lib/src/json_schema/json_schema.dart

```diff
@@ -603,6 +603,7 @@ class JsonSchema {
     JsonSchema firstResult;
     JsonSchema secondResult;
     dynamic firstError;
+    // ignore: unused_local_variable
```

Comment:
Why not remove it? Seems like it's truly unused?

Reply:
Because I would like you to comment on how I can best combine the 2 errors if there are 2 errors. You fell for my trap! (sorta)